Add alignment classes to new site and create update hook for existing.

emerham opened the pull request.

emerham committed:
Add alignment classes to new site and create update hook for existing.

Ruefa commented on osu_standard.install:
```diff
@@ -39,6 +39,15 @@ function osu_standard_install($is_syncing) {
+  $config = \Drupal::service('config.factory')->getEditable('bootstrap_styles.settings');
+  $config->set('items_alignment', $vertical_alignment_options);
+  $config->save();
+}
 /**
```
Need a line between these functions

emerham replied:
Updated

emerham committed:
Updating code standards

emerham committed:
Changed from one update to multiple updates for bootstrap styles.
Needed to remove border styles and some animations. Update width options with more choices.
